polaris-core/src/main/java/org/apache/polaris/core/storage/aws/AwsStorageConfigurationInfo.java

```diff
@@ -122,6 +123,21 @@ public void setRegion(@Nullable String region) {
     this.region = region;
   }
 
+  public String getAwsAccountId() {
```

Comment:
```suggestion
  @JsonIgnore public String getAwsAccountId() {
```
(since this is a derived property)